.env.example

```
@@ -1,11 +1,28 @@
-TELEGRAM_TOKEN=YOUR_TELEGRAM_BOT_TOKEN
-PROJECT_NAME=PromoSectionBot
+# Environment Configuration
+ENVIRONMENT=development  # Options: development, production, test
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)